lf.tf

```diff
@@ -131,3 +137,58 @@ resource "aws_lakeformation_permissions" "catalog_client_system_permissions" {
     wildcard      = true
   }
 }
+
+resource "aws_lakeformation_permissions" "customer_account_permissions" {
+  for_each = var.disable_glue_db_init && var.create_lf_resource ? tomap({
+    for schema in local.customer_account_schemas : "${schema["schema_name"]}-${schema["customer_account"]}" => schema
+  }) : {}
+
+  principal                     = each.value.customer_account
+  permissions                   = ["DESCRIBE"]
```

Comment:
Should we create a variable for permissions? in case we would like to add/remove any?

Reply:
this doesn't allow any data access, which we are planning to add using tag based access, this just allows you to list all tables, to essentially replace hive metastore RO